core-bundle/assets/controllers/image-size-controller.js

```diff
@@ -29,6 +41,30 @@ export default class extends Controller {
         }
     }
 
+    _updateInputs() {
+        const widthInput = this.inputTargets[0];
+        const heightInput = this.inputTargets[1];
+        const select = this.selectTarget;
+        const value = select.value;
+
+        if (value === '' || value.indexOf('_') === 0 || value.toInt().toString() === value) {
+            widthInput.readOnly = true;
+            heightInput.readOnly = true;
+            let dimensions = select.options[select.selectedIndex].text;
+            dimensions =
+                dimensions.split('(').length > 1 ? dimensions.split('(').getLast().split(')')[0].split('x') : ['', ''];
```

Comment:
Did you do this or did Prettier format the line this way? Because I would find the following a lot more readable:

```suggestion
            dimensions = dimensions.split('(').length > 1 ?
                dimensions.split('(').getLast().split(')')[0].split('x') :
                ['', ''];
```

Reply:
Executing `dimensions.split('(')` twice could probably be optimized, too.

Reply:
> Did you do this or did Prettier format the line this way?

I originally copied it as it was from `Backend.enableImageSizeWidgets()`. Prettier then formatted it this way.

Reply:
Instead of doing these string operations, we could also just provide more info in the `config` object we have now for our controller, wdyt?

Reply:
I decided to leave this as is and only optimize `.split('(')`. There isn't really a more straight forward way to get the dimensions in PHP.